tests/xbot/common/subsystems/MockPoseSubsystem.java

```diff
@@ -0,0 +1,34 @@
+package xbot.common.subsystems;
+
+import com.google.inject.Inject;
+
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.XPropertyManager;
+
+public class MockPoseSubsystem extends AbstractPoseSubsystem {
```

Comment:
I'd rather this be called `TestPoseSubsystem`, or something other than "mock". We use the term "mock" to refer to fake entities that aren't legitimately functional.